app/presenters/brexit_taxons_presenter.rb

```diff
@@ -8,21 +8,28 @@ class BrexitTaxonsPresenter
     /education
   ).freeze
 
+  REJECTED_TAXONS = %w(
+    /government/all
+  ).freeze
+
   def featured_taxons
     @featured_taxons ||= FEATURED_TAXONS
-      .map { |base_path| Taxon.find(base_path) }
-      .map
-      .with_index(1) { |taxon, index| BrexitTaxonPresenter.new(taxon, index) }
+      .to_enum
+      .with_index(1)
+      .map do |base_path, index|
```

Comment:
Perhaps you can do `FEATURED_TAXONS.map.with_index(1) do ...` rather than using `to_enum`

Reply:
You still need the `map` after `with_index(1)` as otherwise `@featured_taxons` isn't going to have the correct content (an array of presenters).

As far as I'm aware, `to_enum` and `map` (without any block) are equivalent, and I think it makes the intention of the code more obvious when used as we have above, rather than `map` implicitly returning an enumerable.

Happy to discuss this further.